app/routes.py

```diff
@@ -0,0 +1,292 @@
+from flask import Blueprint, jsonify, make_response
+from flask.globals import request
+from app.models.customer import Customer 
+from app.models.video import Video
+from app.models.rental import Rental, due_date
+from app import db
+from datetime import date, datetime
+
+from tests.test_wave_02 import CUSTOMER_ID
+
+
+# --------------------------------
+# -------- CUSTOMER ROUTES -------
+# --------------------------------
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["POST"])
+def create_customer():
+    request_body = request.get_json()
+
+    missing = ""
+    if "name" not in request_body:
+        missing = "name"
+    elif "postal_code" not in request_body:
+        missing = "postal_code"
+    elif "phone" not in request_body:
+        missing = "phone"
+    if missing:
+        return {"details": f"Request body must include {missing}."}, 400
+
+    new_customer = Customer(
+        name=request_body["name"],
+        postal_code=request_body["postal_code"],
+        phone=request_body["phone"]
+    )
+
+    db.session.add(new_customer)
+    db.session.commit()
+
+    return {"id": new_customer.id}, 201
+
+@customers_bp.route("", methods=["GET"])
+def read_customers():
+    customers = Customer.query.all()
+
+    response_body = []
+
+    for customer in customers:
+        response_body.append(customer.to_dict())
+    
+    return jsonify(response_body), 200
+
+@customers_bp.route("/<id>", methods=["GET"])
+def read_one_customer(id):
+    try:
+        int(id)
+    except:
+        return {"message": "Invalid data"}, 400
+
+    customer = Customer.query.get(id)
+
+    if not customer:
+        return {"message": f"Customer {id} was not found"}, 404        
```

Comment:
Nice logic! Consider extracting this to a helper method so it can be used across your routes.